### DISCUSSION TITLE
Questions regarding Lazer's directory and improting from stable

### DISCUSSION THREAD

**Discussion by nerootter** [ppy/osu#26435] [Q&A] [answered]
While importing my beatmaps from stable, I have found out that it does so to my Windows drive which I don't really want sicne its partitioned to exactly 100GB and my maps are around 70GB (and the fact is, lazer never presents you with an option to change game's destination at setup)

After sucessfully importing all maps, I move my game's files to another directory from lazer's settings menu via respectful option, clean up the thing it tells me to over Appdata, and it does what it says, I guess, but then when I relaunch the game all beatmaps are gone and the first-run set up wizard pops up

How do I exactly move all my stuff? Is there an option to install game somewhere other than C:\?

**Comment by SupDos** [comment]
You don't need to clear anything from appdata - the game's .exe will always be in AppData/Local/osulazer, but this folder shouldn't ever be more than 1GB, and most of the time it'll be around 200MB. 

AppData/Roaming/osu will also leave some files, but they are what tell the game that you've moved the data folder to a different location, so you shouldn't need to clean any of this either

**Comment by peppy** [comment]
It sounds like you probably broke things when you manually "cleaned up". If you still have a copy of the files you cleaned up, try placing them back.

**Reply by nerootter** [reply]
Can you please explain the step-by-step process of migrating from stable to lazer given the following conditions:

My SSD disk is partitioned to 100 gb for C:/ And the rest goes to F:/
My stable is on F:/
My lazer is on C:/
And when I import stuff it goes to C:/ (not exactly i know but still)
And i want to import all maps to lazer and delete stable, !so all 70 gb of mania maps are on F:/ without any traces of them anywhere else!

Thank you in advance and sorry for any misunderstandings

**Reply by SupDos** [reply, marked as answer]
- Install lazer
- Open lazer, ignore first time setup (press ESC)
- Go to settings, "Change folder location" to a folder on your F Drive
- Once that has completed, "Run setup wizard" in settings and go to the Import page
- If it hasn't detected it already, locate your stable install
- Import your files

**Reply by nerootter** [reply]
Can (and how, via Apps & Features or manually) I simply delete stable afterwards? Where will the beatmaps be stored?

**Reply by SupDos** [reply]
Yes, you can delete stable after importing, lazer stores files seperately. Either uninstall stable through Settings/Control Panel, or delete the osu folder for stable manually

Lazer will store its files in whatever place you picked when Changing folder location

**Reply by nerootter** [reply]
Just did all the steps, it worked!!! Thank you all :) Hyped for lazer now
